Example/.env

```diff
@@ -0,0 +1,2 @@
+ClientIdDebug = "MRC6bwFASOm0sJYPYwVEt2KD51bkAtdgS7ltqeDYUf4"
```

Comment:
Please add this one in gitignore, and create a file `.env.example`
```
ClientIdDebug = ""
ClientSecretDebug = ""
```

Reply:
Updated in 92249d76